community-operators/kubeturbo/kubeturbo-operator.v6.3.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[{"apiVersion":"charts.helm.k8s.io/v1alpha1","kind":"Kubeturbo","metadata":{"name":"kubeturbo-example"},"spec":{"restAPIConfig":{"opsManagerPassword":"Turbo_password","opsManagerUserName":"Turbo_username"},"serverMeta":{"turboServer":"https://Turbo_server_URL","version":6.3}}}]'
+    capabilities: Basic Install
+  name: kubeturbo-operator.v6.3.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: Kubeturbo
+      name: kubeturbos.charts.helm.k8s.io
+      version: v1alpha1
+  description: Placeholder description
```

Comment:
Please provide a valid description for the operator.

Reply:
of course, done